src/translations/en.json

```diff
@@ -783,13 +789,18 @@
             }
           },
           "unavailable": "This entity is unavailable.",
-          "enabled_label": "Enable entity",
+          "view_status": "Entity view status",
+          "enabled_label": "Enabled",
+          "disabled_label": "Disabled",
           "enabled_cause": "Disabled by {cause}.",
+          "hidden_label": "Hidden",
+          "hidden_cause": "Hidden by {cause}.",
           "device_disabled": "The device of this entity is disabled.",
           "open_device_settings": "Open device settings",
           "enabled_description": "Disabled entities will not be added to Home Assistant.",
           "enabled_delay_confirm": "The enabled entities will be added to Home Assistant in {delay} seconds",
           "enabled_restart_confirm": "Restart Home Assistant to finish enabling the entities",
+          "hidden_description": "Hidden entities will not be shown in Home Assistant UI.",
```

Comment:
```suggestion
          "hidden_description": "Hidden entities will not be shown. Their history is still tracked and they can still be targeted with services.",
```

Reply:
How about `Hidden entities will not be shown on your dashboard. Their history is still tracked and you can still interact with them with services.`

Reply:
I'm no longer using this but I guess we should add it back

Reply:
Can we use it here?

![image](https://user-images.githubusercontent.com/1444314/158697294-5ad618c5-4cb9-41b2-9cf8-24f2b9fc08f6.png)

Reply:
Thats the disabled description not hidden